src/Symfony/Component/Security/Http/RememberMe/PersistentRememberMeHandler.php

```diff
@@ -55,7 +55,7 @@ public function createRememberMeCookie(UserInterface $user): void
     {
         $series = base64_encode(random_bytes(64));
         $tokenValue = $this->generateHash(base64_encode(random_bytes(64)));
-        $token = new PersistentToken(\get_class($user), method_exists($user, 'getUserIdentifier') ? $user->getUserIdentifier() : $user->getUsername(), $series, $tokenValue, new \DateTime());
+        $token = new PersistentToken(method_exists('getUserIdentifier') ? $user->getUserIdentifier() : $user->getUsername(), $series, $tokenValue, new \DateTime());
```

Comment:
The `method_exists()` call is broken now.

Reply:
I get that $user is in fact still needed in construct